addons/mail/static/src/xml/chatter.xml

```diff
@@ -60,7 +60,7 @@
                 <!-- List of followers -->
                 <div class="o_composer_suggested_partners">
                     <t t-foreach='widget.suggestedPartners' t-as='recipient'>
-                        <div t-attf-title="Add as recipient and follower (reason: #{recipient.reason})">
+                        <div t-attf-title="Add as recipient and follower (reason: #{recipient.reason})" t-att-class="recipient_index &gt; 2 ? 'o_suggested_toggle o_suggested_hidden' : ''">
```

Comment:
I don't think you need to escape `>` to `&gt;`

Reply:
undefined parameter `recipient_index` in the template

Reply:
&gt is needed or crash, `recipient_index` is generated by foreach i think, it works fine